<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>678</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Simple Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_statoInverter">
      <property name="title">
       <string>Stato Inverter</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_init">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>INIT</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_freqEst">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>FREQ ESTIM.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_readyToOp">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>READY TO OP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fault">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>FAULT</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_opEnabled">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>OP. ENABLED</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Active Faults:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_OVER_VOLTAGE">
           <property name="text">
            <string>OVER_VOLTAGE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SYNCH_SCR_LOST">
           <property name="text">
            <string>SYNCH_SCR_LOST</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_FREQ_SCR_UNDEF">
           <property name="text">
            <string>FREQ_SCR_UNDEF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_UNDER_VOLTAGE">
           <property name="text">
            <string>UNDER_VOLTAGE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_DESAT">
           <property name="text">
            <string>DESAT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_PSUPLLY">
           <property name="text">
            <string>PSUPLLY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_EXTERNAL_FLT">
           <property name="text">
            <string>EXTERNAL_FLT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_OVER_CURRENT">
           <property name="text">
            <string>OVER_CURRENT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_UNDER_CURRENT">
           <property name="text">
            <string>UNDER_CURRENT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_OH1">
           <property name="text">
            <string>OH1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_OH2">
           <property name="text">
            <string>OH2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_controlloInverter">
      <property name="title">
       <string>Controllo Inverter</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_start">
         <property name="text">
          <string>Start / Off</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_ackFault">
         <property name="text">
          <string>Ack Fault</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_ImpulsiSpenti">
         <property name="text">
          <string>Impulsi
Spenti</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_impulsiAttivi">
         <property name="text">
          <string>Impulsi
Abilitati</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_process"/>
    </item>
    <item row="0" column="0" rowspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_params">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
    <addaction name="actionOpen_Log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="bottomStatusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionClear"/>
   <addaction name="actionAutoTrigger"/>
   <addaction name="actionSynchronizeParams"/>
   <addaction name="actionWrite_Param"/>
   <addaction name="actionRead_All"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/Plugged_16x.png</normaloff>:/images/Plugged_16x.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/Unplugged_16x.png</normaloff>:/images/Unplugged_16x.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-settings-32.png</normaloff>:/images/icons8-settings-32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-broom-32.png</normaloff>:/images/icons8-broom-32.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_Log">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="actionSynchronizeParams">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-synchronize-32.png</normaloff>:/images/icons8-synchronize-32.png</iconset>
   </property>
   <property name="text">
    <string>Synchronize Params</string>
   </property>
   <property name="toolTip">
    <string>Write Params And Read All</string>
   </property>
  </action>
  <action name="actionAutoTrigger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-agopressione-50.png</normaloff>:/images/icons8-agopressione-50.png</iconset>
   </property>
   <property name="text">
    <string>AutoTrigger</string>
   </property>
   <property name="toolTip">
    <string>Istantaneous Write Params</string>
   </property>
  </action>
  <action name="actionWrite_Param">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-scaricare-50.png</normaloff>:/images/icons8-scaricare-50.png</iconset>
   </property>
   <property name="text">
    <string>Write Param</string>
   </property>
   <property name="toolTip">
    <string>Write Selected Parameter</string>
   </property>
  </action>
  <action name="actionRead_All">
   <property name="icon">
    <iconset resource="../IKE_gui.qrc">
     <normaloff>:/images/icons8-caricare-50.png</normaloff>:/images/icons8-caricare-50.png</iconset>
   </property>
   <property name="text">
    <string>Read All</string>
   </property>
   <property name="toolTip">
    <string>Read Parameters and Process Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../IKE_gui.qrc"/>
 </resources>
 <connections/>
</ui>
